agent/src/beerocks/slave/gate/unit_test/gate_test.cpp

```diff
@@ -118,4 +122,53 @@ TEST(gate_beacon_query_test, loading_vs_specific_from_1905_beacon_query)
     ASSERT_EQ(beerocks_params.use_optional_ssid, 0);
 }
 
+/////// vs --> 1905 ////////////////////////////////////////////////////////////////////////////////////
+
+/**
+* const values to be used for both assignment and test
+*/
+const auto associated_sta_mac_2 = sMacAddr{0xba, 0xd0, 0xde, 0xad, 0xbe, 0xef};
+
+void fill_vs_values(beerocks_message::cACTION_MONITOR_CLIENT_BEACON_11K_RESPONSE &vs)
+{
+    vs.params().sta_mac = associated_sta_mac_2;
+}
+
+TEST(gate_beacon_response_test, loading_1905_from_vs_response)
+{
+    // preperation for beerocks
+
+    // class list
+    constexpr const uint16_t BUF_SIZE_BEEROCKS = 10000;
+    uint8_t buff_beerocks[BUF_SIZE_BEEROCKS];
+    ClassList cl(buff_beerocks, BUF_SIZE_BEEROCKS);
+
+    // beerocks
+    cl.addClass<beerocks_message::cACTION_MONITOR_CLIENT_BEACON_11K_RESPONSE>();
```

Comment:
Same comments here than in previous test about checking returned values and constexpr + const